### PR TITLE
Add new "Song Contest" category for CreateWiki

### PR DESCRIPTION
**Pull request by R4356th**
We get a lot of requests for wikis related to song contests these days. A category dedicated to those would be great.

### CODE REVIEW

File: LocalSettings.php [hunk 1]
```diff
@@ -727,6 +727,7 @@
 			'Religion' => 'religion',
 			'Science' => 'science',
 			'Software/Computing' => 'software',
+			'Song Contest' => 'sc',
 			'Sports' => 'sport',
```
Comment (Universal-Omega) [inline LocalSettings.php:731]: Why use just `sc`? Why not songcontest?

**Review by Universal-Omega — CHANGES_REQUESTED** [review]

File: LocalSettings.php (hunk 1 above)
Comment (R4356th) [reply LocalSettings.php:731]: To save time. There are two other examples above.

Comment (Universal-Omega) [reply LocalSettings.php:731]: This is not as descriptive though and that's because those include special characters that I don't think can be in this.

Comment (R4356th) [reply LocalSettings.php:731]: Done.

Comment (Universal-Omega) [reply LocalSettings.php:731]: Thanks!

**Review by Universal-Omega — APPROVED** [review]

**Comment by dmehus** [comment]
We've been using Music for these wikis. Given how many existing song contest wikis are categorized as _music_ already, I personally feel as though we should merge _Song Contest_ category into the _Music_ category, and possibly rename it as either _Music (including Song Contest wikis)_ or _Music & Song Contests_. Otherwise, we'll end up (a) fragmenting how wikis are categorized, reducing the utility to the categorization scheme in WikiDiscover, (b) creating too many narrow categories, and/or (c) both.

**Comment by R4356th** [comment]
> We've been using Music for these wikis. Given how many existing song contest wikis are categorized as _music_ already, I personally feel as though we should merge _Song Contest_ category into the _Music_ category, and possibly rename it as either _Music (including Song Contest wikis)_ or _Music & Song Contests_. Otherwise, we'll end up (a) fragmenting how wikis are categorized, reducing the utility to the categorization scheme in WikiDiscover, (b) creating too many narrow categories, and/or (c) both.

I think the number of requests for this kind of wikis justifies the addition.

**Comment by dmehus** [comment]
> > We've been using Music for these wikis. Given how many existing song contest wikis are categorized as _music_ already, I personally feel as though we should merge _Song Contest_ category into the _Music_ category, and possibly rename it as either _Music (including Song Contest wikis)_ or _Music & Song Contests_. Otherwise, we'll end up (a) fragmenting how wikis are categorized, reducing the utility to the categorization scheme in WikiDiscover, (b) creating too many narrow categories, and/or (c) both.
> 
> I think the number of requests for this kind of wikis justifies the addition.

Well, I_do_ think it would still be better to merge the two categories into the original Music category, given how many song contest wikis are categorized in this way. Ideally, it'd be nice to have a nested wiki category system with broader and narrower categories, but since we have one level of category, we should use a broader categorization scheme as we've always done, as I _do_ know that Reception123 is hesitant to add too many narrowly focused categories.

I _do_ agree with you, though, that given the number of song contest wikis we have, we could add a parenthetical qualifier to the **Music** category that notes this category includes song contest wikis.

**Pull request merged**